utilities/simulator_cache/cache_simulator_test.cc

```diff
@@ -0,0 +1,333 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "utilities/simulator_cache/cache_simulator.h"
+
+#include <cstdlib>
+#include "db/db_test_util.h"
+
+namespace rocksdb {
+namespace {
+const std::string kBlockKeyPrefix = "test-block-";
+const std::string kRefKeyPrefix = "test-get-";
+const uint64_t kGetId = 1;
+const uint64_t kGetBlockId = 100;
+const uint64_t kCompactionBlockId = 1000;
+const uint64_t kCacheSize = 1024 * 1024 * 1024;
+const uint64_t kGhostCacheSize = 1024 * 1024;
+}  // namespace
+
+class CacheSimulatorTest : public DBTestBase {
```

Comment:
Looks like the base class does not have to be `DBTestBase` since many of its database operations, e.g. `Open`, `Close`, `Put`, `Flush`, etc. are not needed. Maybe consider use `testing::Test` or `testing::WithParamInterface`?

Reply:
Thanks. I changed to testing::Test.